src/js/labels/type.ts

```diff
@@ -0,0 +1,17 @@
+/* Smaller label object for representing labels in a list */
+export type LabelNested = {
+    /* A hex encoded color */
+    color: string;
+    /* A detailed description */
+    description: string;
+    /* A unique identifier */
+    id: number;
+    /* A name for the label */
```

Comment:
No need to specify relation to the label in the docstring. Doing so is redundant since the relation is inferred due to the `name` being a field of the model